lax/lichens/sciencerun0.py

```diff
@@ -249,3 +251,42 @@ def _process(self, df):
             df.loc[:, self.__class__.__name__] = (S2Width.relative_s2_width_bounds(df.s2,
                                                                             kind='low') <= df.temp)
             return df
+
+
+
+class S1SingleScatter(Lichen):
+    """
+    Requires only one valid interaction between the largest S2, and any S1 recorded before it.
+
+    The S1 cut checks that any possible secondary S1s recorded in a waveform, could not have also
+    produced a valid interaction with the primary S2. To check whether an interaction between the 
+    second largest S1 and the largest S2 is valid, we use the S2Width cut. If the event would pass
+    the S2Width cut, a valid second interaction exists, and we may have mis-identified which S1 to
+    pair with the primary S2. Therefore we cut this event. If it fails the S2Width cut the event is
+    not removed.
+
+    Current version is developed on unblinded Bkg data (paxv6.4.2). It is described in this note:
+	https://xecluster.lngs.infn.it/dokuwiki/doku.php?id=xenon:xenon1t:jacques:s1_single_scatter_cut
+
+    It should be applicable to data regardless of if it is ER or NR.
+
+    Author: Jacques (jpienaa@purdue.edu).
+    """
+
+    version=0
+
+    def process(self, df):
+
+        s2width=S2Width()
```

Comment:
Why do you have to instantiate this if it's a static method above?  Also, try not to overload process but instead _process.    atl_rel_width should get loaded into Temp.